test/previews/primer/alpha/nav_list_preview.rb

```diff
@@ -0,0 +1,364 @@
+# frozen_string_literal: true
+
+module Primer
+  module Alpha
+    # @label NavList
+    class NavListPreview < ViewComponent::Preview
+      def playground
```

Comment:
Can you add params that NavList has to this `Playground` preview, and also add a `Default` preview with a simple example?

I like the example with show more. Can you delete all the others copied from ActionList?

Reply:
If I recall correctly, we decided to rename this playground preview to "default" and not expose any params, since the preview renders an entire list + sub items.

Yes, I have deleted the other previews copied from ActionList 😄

Reply:
Yup! Give me a nice capitol `Default` for my mental health please 😆